DM-1821 SmartRest static templates 123 and 124. (#826)
SoftwareAG/c8y-docs: content/reference/smartrest-two-bundle/mqtt-static-templates.md

+ <a href="#120">120,configType,url,filename[,time]</a>
+ <a href="#121">121,profileExecuted,profileID</a>
+ <a href="#122">122,agentName,agentVersion,agentURL,agentMaintainer</a>

> + <a href="#123">123</a>

Reviewer: ```suggestion
+ <a href="#123">123 (Get internal ID, reply: 124,id)</a>
```

140,docker,3.2.1,systemd,https://www.docker.com/,nginx,1.6,container,https://www.nginx.com/
```

<a name="123"></a>

> ##### Get device managed object id (123)

Reviewer 2: ```suggestion
##### Get the device managed object ID (123)
```

> Trigger sending of the id of the device managed object.

Reviewer 2: ```suggestion
Retrieve the ID of the device managed object.
```

106,child1,child2,child3
```

<a name="124"></a>

> ##### Get device managed object id (124)

Reviewer 2: ```suggestion
##### Get the device managed object ID (124)
```

> Gets the id of the device managed object.

Reviewer 2: ```suggestion
Retrieve the ID of the device managed object.
```

> Retrieve the ID of the device managed object.

Reviewer 3: I think this is called in the documentation `global ID` or `internal ID`. @rssag am I correct?

You can consider mentioning that it is based on serial of "current device" or "current connection", but maybe it is obvious or can be mentioned in 123.

Author: Is it based on the serial? I don't know this. How do you know?

Reviewer 3: From the implementation. If I understand it correctly you are connecting device via MQTT, with some ClientId, and sending `123` message on static smartREST topic. 

Device ID (GId) is resolved in all such scenarios based on `c8y_Serial` which exactly fits `ClientId` header from mqtt client (or Managed Object is created if nothing found). This is how you can create all MEA stuff for a device without knowing its GID. I think your new feature just retrieves this GId and return it in `124` message.

@l2p- am I correct or missunderstand something?

Reviewer 2: there's something about a global identifier here:
https://cumulocity.com/guides/concepts/domain-model/#object-identification

in the inventory API, managed objects just have a field "id" for the ID though.

|Position|Parameter|Type|
|:-------|:-------|:---|
|1|id|String|

**Example**

```text

> 106,id

Reviewer: ```suggestion
124,12345
```
We use real examples in other places instead of placeholders + typo in the mesage ID

Author: Thanks
